src/main/java/org/spongepowered/common/event/tracking/phase/packet/PacketPhase.java

```diff
@@ -93,91 +93,48 @@
 
     public static final class General {
 
-        public static final IPacketState UNKNOWN = new UnknownPacketState();
         public static final IPacketState MOVEMENT = new MovementPacketState();
         public static final IPacketState INTERACTION = new InteractionPacketState();
         public static final IPacketState IGNORED = new IgnoredPacketState();
         public static final IPacketState INTERACT_ENTITY = new InteractEntityPacketState();
         public static final IPacketState ATTACK_ENTITY = new AttackEntityPacketState();
         public static final IPacketState INTERACT_AT_ENTITY = new InteractAtEntityPacketState();
-        public static final IPacketState CHAT = new ChatPacketState();
```

Comment:
A bunch of them were just `BasicPacketState`s; I used `IGNORED` instead. `UNKNOWN` was merged into `HANDLED_EXTERNALLY`.